### DISCUSSION TITLE
Limit size of database

### DISCUSSION THREAD

**Discussion by MarkMD0OMY** [chuot/rdio-scanner#425] [Q&A] [answered]
Hi All,
Is there a way to limit the size / amount of recordings on the Search Call button?
I don't want it filling up my disk and unless I'm being blind, I don't see an option on the Admin page...?

Thanks

**Comment by atlex** [comment, marked as answer]
@MarkMD0OMY There is an option in the admin page to prune records after N days. But it doesn't work very well for me. Some day I have a lot of entries and the database becomes so huge.

**Reply by MarkMD0OMY** [reply]
Thanks for that. I found the "Prune Days" setting in the Options on the Admin page so I've changed that.
Also noticed something else but will start a new thread for that

Thanks for the help
